### ISSUE TITLE
Learning rate scaling in linear.py

### ISSUE THREAD

**Issue by MohammedSB** [facebookresearch/dinov2#155]
Hello,

So from my understanding, all the learning rates for the different classifiers tried when running linear.py are scaled using the funtion below.
```
def scale_lr(learning_rates, batch_size):
    return learning_rates * (batch_size * distributed.get_global_size()) / 256.0
```

I am specifically trying to understand why are the lrs divided by 256 here, and not some number that varies with the batch_size/global batch size? I want to train using 1 GPU on a single node, using a somewhat smaller batch size (32-64) than the default on linear.py, so I am trying to understand how/if I should change this for my setting.

**Comment by qasfb** [comment]
Essentially this is for legacy reasons; the original DINO work was using a reference learning rate value adapted for batch size 256, which was scaled afterwards. If this is posing a problem, I recommend using a logistic regression classifier on the [CLS] tokens (it should also be faster to train if the number of samples involved and memory allow), it gives a performance only around 0.2-0.3 % lower usually.

**Comment by MohammedSB** [comment]
Well, I already adapted the linear.py code to my own purpose; basically I'll be doing multilabeled classification where each neuron in the linear layers acts as a logistic regression model, by applying sigmoid instead softmax and calculating the class-specific performance metrics from there. 

I just wanted to see how I can adjust the learning rate scaler, I'll try slowly reducing that 256.0 number and see how it compares, but advice on how I can change that would very much be appreciated and save me some time.

**Comment by qasfb** [comment]
The linear.py code should already be performing a grid on many learning rates; it does one forward of the backbone, then trains multiple linear classifiers with {1,4} last cls tokens and with/without the avg pool of the patch tokens.

You also have a setting to provide your learning rates (before scaling) there:
https://github.com/facebookresearch/dinov2/blob/main/dinov2/eval/linear.py#L95-L100
